src/Symfony/Component/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php

```diff
@@ -81,7 +81,7 @@ public function handleRequest(FormInterface $form, $request = null)
                 $form->submit(null, false);
 
                 $form->addError(new FormError(
-                    $form->getConfig()->getOption('post_max_size_message'),
+                    (string) $form->getConfig()->getOption('post_max_size_message'),
```

Comment:
just checked and I found that it can lead a bc break (if someone try to serialize this option)